packages/example-codehub/src/codehub-application.ts

```diff
@@ -3,16 +3,20 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application, CoreBindings} from '@loopback/core';
+import {Application} from '@loopback/core';
+import {RestServer, RestBindings} from '@loopback/rest';
 import {UserController, HealthController} from './controllers';
 
 export class CodeHubApplication extends Application {
   constructor() {
     super();
 
     const app = this;
+    app.server(RestServer);
+    // We can't use the async version in the ctor!
+    const server = app.getServerSync(RestServer);
```

Comment:
Ouch! We cannot rely on sync getter in production code. What if the RestServer is depending on configuration that requires async computation? For example, what if we are fetching the config from Consul?

Reply:
I've changed it to use the async function, but now I'm using a floating Promise instead, which still feels icky (I had to suppress the tslint warning for it).